test/unit/ngsiv2/HTTP_reveice_measures-test.js

```diff
@@ -0,0 +1,329 @@
+                method: 'POST',
+                json: {
+                    humidity: '111222'
+                },
+                headers: {
+                    'fiware-service': 'smartGondor',
+                    'fiware-servicepath': '/gardens'
+                },
+                qs: {
+                    i: 'dev0130101',
+                    k: '1234',
+                    t: '20200222T222222'
+                }
+            },
+            provisionOptions = {
+                url: 'http://localhost:' + config.iota.server.port + '/iot/devices',
+                method: 'POST',
+                json: utils.readExampleFile('./test/deviceProvisioning/provisionDeviceTimeinstant.json'),
+                headers: {
+                    'fiware-service': 'smartGondor',
+                    'fiware-servicepath': '/gardens'
+                }
+            };
+
+        beforeEach(function(done) {
+            nock.cleanAll();
+
+            contextBrokerMock = nock('http://192.168.1.1:1026')
+                .matchHeader('fiware-service', 'smartGondor')
+                .matchHeader('fiware-servicepath', '/gardens')
+                .post('/v1/updateContext')
+                .reply(200, utils.readExampleFile('./test/contextResponses/multipleMeasuresSuccess.json'));
+
+            contextBrokerMock
+                .matchHeader('fiware-service', 'smartGondor')
+                .matchHeader('fiware-servicepath', '/gardens')
+                .post('/v2/entities/e0130101/attrs',
+                    utils.readExampleFile('./test/unit/ngsiv2/contextRequests/timeInstantMeasures.json'))
+                .reply(204);
+
+            iotagentMqtt.stop(function() {
+                config.iota.timestamp = true;
+                config.compressTimestamp = false;
+                iotagentMqtt.start(config, function() {
+                    request(provisionOptions, function(error, response, body) {
+                        done();
+                    });
+                });
+            });
+        });
+
+        afterEach(function() {
+            config.iota.timestamp = false;
+            config.compressTimestamp = true;
+        });
+
+        it('should send its value to the Context Broker', function(done) {
+            request(optionsMeasure, function(error, result, body) {
+                contextBrokerMock.done();
+                done();
+            });
+        });
+    });
+
+    describe('When a POST measure arrives for an unprovisioned device', function() {
+        var optionsMeasure = {
+            url: 'http://localhost:' + config.http.port + '/iot/json',
+            method: 'POST',
+            json: {
+                humidity: '32',
+                temperature: '87'
+            },
+            headers: {
+                'fiware-service': 'smartGondor',
+                'fiware-servicepath': '/gardens'
+            },
+            qs: {
+                i: 'MQTT_UNPROVISIONED',
```

Comment:
HTTP_UNPROVISIONED ?

Reply:
As it happens with https://github.com/telefonicaid/iotagent-json/pull/265#pullrequestreview-90500209, I understand that this refers to the IOTA and no to the transport binding. I would rather use JSON_UNPROVISIONED.

Reply:
Fixed in e2760b3aee7d517d554809fed0e2f7f1a2102a13